docs/config.md

```diff
@@ -72,6 +72,17 @@ All options can be overridden in `mango.local.yaml` (or `mango.local.json`) file
 
 ---
 
+### Images
+
+* `images` - array of all images - element can be:
+    * string (glob) of source image for minification or
+    * object for resize
+        * `src` - string (glob) source of images
+        * `sizes` - array of widths (int)
+        * `aspectRatio` - aspect ratio of image on output (float = width/height)
```

Comment:
To je obecně dost neřešitelný, neni? Buď můžeš udělat crop, kde je pak potřeba nadefinovat kde má být ten anchor, nebo to může udělat lettering. Nebo tam můžou být nějaké defaults, ale to je takové divné.